src/compas_ags/rhino/forceobject.py

```diff
@@ -60,6 +60,7 @@ def scale_from_3_points(self):
         # get the first reference point
         gp = Rhino.Input.Custom.GetPoint()
         gp.SetCommandPrompt('Select the first reference point.')
```

Comment:
"Select the start point of the reference line."

Reply:
Hi Tom.. I name all these points in this way: 
scale from 3 points: 1: "base point", 2: "1st reference pt", 3: "2nd reference pt"
scale from 4 points: 1: "start pt of 1st reference line", 2: "end pt of 1st reference line", 3: "start pt of 2nd reference line", 4: "end pt of 2nd reference line